firmware/thruna/DCM/Src/Io/Io_EllipseImu.c

```diff
@@ -291,6 +416,12 @@ float Io_EllipseImu_GetSensorOutput(EllipseImuOutput output)
     return *(sensor_output_map[output]);
 }
 
+float Io_EllipseGps_GetSensorOutput(EllipseImuOutput output)
+{
+    assert(output < NUM_SBG_OUTPUTS);
+    return *(sensor_output_map[output]);
+}
```

Comment:
What is this function for?

Reply:
This just checks that you are trying too access a valid member of the enum (since NUM_SBG_OUTPUTS is the last index in  the enum)